server/controllers/demo.js

```diff
@@ -0,0 +1,55 @@
+require("dotenv").config();
+const User = require("../models/User");
+const mongoose = require("mongoose");
+const generateToken = require("../utils/generateToken");
+
+exports.populate = async (_, res) => {
+  if (
+    process.env.NODE_ENV !== "production" &&
+    mongoose.connection.readyState === 1
+  ) {
+    console.log("Seeding database...");
+    mongoose.connection.dropDatabase();
```

Comment:
One concern, in production this is very dangerous. Do we need to delete the entire database or could we just delete the demo user itself? Let me know what you're thoughts are on this.

Reply:
You are very right, I thought the demo user wont make it to production. It will be better to delete that particular user.

Reply:
I  don't think we need to reset the demo user. I think I get the scope better now